datalad/interface/run_procedure.py

```diff
@@ -33,6 +35,7 @@
 from datalad.distribution.dataset import datasetmethod
 from datalad.support.exceptions import InsufficientArgumentsError
 from datalad.support.exceptions import NoDatasetArgumentFound
+from datalad.utils import on_windows, maybe_shlex_quote
```

Comment:
It is not used consistently in the code base (yet), but we now do (for any import with more than a single item)

```
from datalad.utils import (
    on_windows,
    maybe_shlex_quot,
)
```

i.e. one per line, each line (incl last one has a trailing comma) -- this minimizes the diffs and make them quicker to grasp.

That being said `on_windows` is no longer needed, see below.

Reply:
Ah yes, I missed this. I removed all superfluous imports now.